packages/postgraphile-core/__tests__/integration/__snapshots__/queries.test.js.snap

```diff
@@ -374,47 +374,47 @@ Object {
     "i": Object {
       "edges": Array [
         Object {
-          "cursor": "WyJjb25zdGFudF9hc2MiLFsyXV0=",
+          "cursor": "WyJjb25zdGFudF9hc2MiLDFd",
```

Comment:
This is a view without a unique ordering so its cursor has changed. This one went from `["constant_asc",[2]]` to `["constant_asc",1]`, `1` being the `row_number()`